packages/nextjs/scaffold.config.ts

```diff
@@ -25,6 +25,7 @@ const scaffoldConfig = {
     chains.zkSyncTestnet,
     chains.scroll,
     chains.scrollSepolia,
+    chains.hardhat,
```

Comment:
Ohh actually you know if we did this, previously the rainbhow mobile wallet would failed and hence we removed it in #86 

But I just tested the deployed version with mobile rainbow wallet and everything seems to work nicely they might have updated few things internally